starlette_admin/views.py

```diff
@@ -667,18 +670,62 @@ def can_view_details(self, request: Request) -> bool:
         """Permission for viewing full details of Item. Return True by default"""
         return True
 
+    def can_view_list(self, request: Request) -> bool:
+        """Permission for viewing list of Items. Return True by default"""
+        return True
+
     def can_create(self, request: Request) -> bool:
         """Permission for creating new Items. Return True by default"""
         return True
 
+    def can_save_and_add_another(self, request: Request) -> bool:
+        """Permission for saving and adding another Items. Returns `can_create` by default"""
+        return self.can_create(request)
+
     def can_edit(self, request: Request) -> bool:
         """Permission for editing Items. Return True by default"""
         return True
 
+    def can_save_and_continue_editing(self, request: Request) -> bool:
+        """Permission for saving and continue editing Items. Returns `can_edit` by default"""
+        return self.can_edit(request)
+
     def can_delete(self, request: Request) -> bool:
         """Permission for deleting Items. Return True by default"""
         return True
 
+    def get_detail_card_title(self, request: Request, obj: Any) -> str:
+        """Return the title of the detail card"""
+        return gettext("Details of #%(pk)") % {"pk": obj[self.pk_attr]}
```

Comment:
I changed the text for this area, I can replace this line as below, it'll be like old days...

```suggestion
        return gettext("#%(pk)") % {"pk": obj[self.pk_attr]}
```